src/classes/Client.ts

```diff
@@ -80,8 +80,18 @@ export default class Client {
 	}
 
 	/** Get playlist information and its videos by playlist id or URL */
-	async getPlaylist(playlistIdOrUrl: string): Promise<Playlist | undefined> {
+	async getPlaylist(playlistIdOrUrl: string): Promise<Playlist | MixPlaylist | undefined > {
 		const playlistId = getQueryParameter(playlistIdOrUrl, "list");
+		if (playlistId.startsWith('RD')) {
```

Comment:
The code doesn't seems to be linted properly, this supposed to be double quote. Please run `npm run lint:fix` to lint the code